src/client/pages/Checkout/CheckoutPayment/CheckoutPayment.tsx

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { useFormik, FormikHelpers } from 'formik'
 import { GraphQLError } from 'graphql'
 import { useTextKeys } from 'utils/textKeys'
+import { useQuoteCartData } from 'utils/hooks/useQuoteCartData'
```

Comment:
Add a new import but no usage of `useQuoteCartData`?

Reply:
Will fix! :D